docs/guides/DolphinPay.md

```diff
@@ -1,216 +1,245 @@
-# 🐬 Try MantaPay in Dolphin Testnet
+# 🐬 Try MantaPay on Dolphin Testnet
 
-MantaPay allows users to privatize public assets to private, transfer private asset, and convert private asset back to public assets (as demonstrated below).
+[The Dolphin zkTransact page](https://app.manta.network/dolphin/transact) allows users to privatize public assets to private, transfer private asset, and convert private asset back to public assets (as demonstrated below).
 
 <div style={{textAlign: 'center'}}>
     <img alt="DolphinPay Overview" src="/img/guides/DolphinPay.svg" width="70%"/>
 </div>
 
-Note: Public assets and private assets have different addresses (public addresses vs shielded addresses). There is no relationship between a public address and a shielded address.
+Note: Public assets and private assets have different addresses (public addresses vs zkAddresses). There is no relationship between a public address and a zkAddress.
 
 ## Setup Testnet Accounts
 
-### Install a Polkadot.JS Browser Extension and Create a Public Address
+### Install a Browser Wallet and Create a Public Address
 
-Currently, Dolphin Testnet V2 supports Polakdot.JS browser extension wallet to manage public assets.
+Currently, Dolphin Testnet V3 supports Talisman, SubWallet, Polakdot.js wallets to manage public assets.
 
-[Download *polkadot.js* browser wallet](https://polkadot.js.org/extension/), open the extension, and create a new account.
+To get started, click the "Connect Wallet" button.
 
-### Install [Manta Signer](https://signer.manta.network) and Create a Shielded Address
-
-Signer is a native app to turbo-charge zero-knowledge proof generation. Signer is available for:
-   * macOS (version 10.5 or later)
-   * Ubuntu (version 18.04 or 20.04)
-   * Windows (version 10 or later)
-
-When you first open Signer, follow the prompts to create a new account:
 <br/>
 
    <div style={{textAlign: 'center'}}>
-    <img alt="signer-init" src="/img/guides/signer-init.png" width="50%"/>
+    <img alt="signer-init" src="/img/guides/connect_wallet_button.png" width="80%"/>
    </div>
 
 <br/>
 
-### Get Testnet Tokens
+If you don't already have a wallet installed, click the "Install" link for your your chosen wallet. Once you install the wallet, it will give you instructions to set up your first account. When you are finished, refresh the page.
```

Comment:
after "refresh the page", I think it will be great to add a little more, "When you are finished, refresh the page and the button text will change from Install to Connect"

Reply:
fixed